lib/ez_gen/template.rb

```diff
@@ -0,0 +1,139 @@
+class Template
+  class << self
+    def rake_template
+      <<~HEREDOC
```

Comment:
Thanks for pointing me in this direction to find the `<<~HEREDOC` syntax for multiline strings. This is definitely much better than how we had it!